engines/bastion_katello/app/assets/stylesheets/bastion_katello/bastion_katello.scss

```diff
@@ -25,5 +25,7 @@
     overflow: hidden;
     text-overflow: ellipsis;
   }
-
+  .description {
```

Comment:
I wonder if we should use a more descriptive name here for the intent in case this class is desired on other, description columns.